adafruit_mcp2515/__init__.py

```diff
@@ -604,9 +636,11 @@ def _get_tx_buffer(self):
         return tx_buffer
 
     def _set_baud_rate(self):
+        # ******* set baud rate ***********
+        if self._crystal_freq not in (16000000, 8000000):
+            raise RuntimeError("Incorrect xtal frequency")
```

Comment:
Can you update the error to match the `__init__()` option?  Also, this seems like a good place for a `ValueError` .  Thanks!

```suggestion
            raise ValueError(f"Incorrect crystal frequency - must be one of: {tuple(_BAUD_RATES.keys())}")
```

Reply:
Done on last commit.